Framework/DataHandling/src/LoadNXSPE.cpp

```diff
@@ -139,6 +139,8 @@ void LoadNXSPE::exec() {
     file.openData("psi");
     file.getData(temporary);
     psi = temporary.at(0);
+    if (std::isnan(psi))
```

Comment:
I would be tempted to say we should post a warning about this. That way we are being transparent that we have not read from the file exactly as it was written.